gax/src/test/java/com/google/api/gax/batching/BatcherImplTest.java

```diff
@@ -251,9 +251,9 @@ public void testWhenElementBytesExceeds() throws Exception {
   public void testWhenThresholdIsDisabled() throws Exception {
     BatchingSettings settings =
         BatchingSettings.newBuilder()
-            .setElementCountThreshold(0)
-            .setRequestByteThreshold(0)
-            .setDelayThreshold(Duration.ofMillis(1))
+            .setElementCountThreshold(null)
```

Comment:
For my understanding, what is the difference between 0 and null? What is the behavior if you don't call any of these setters?

Reply:
Behaviorwise there is no difference, Both of these values are to disable the thresholds.

Earlier we were using [v2.BatchingSettings](https://github.com/googleapis/gax-java/blob/master/gax/src/main/java/com/google/api/gax/batching/v2/BatchingSettings.java#L123-L132) which had primitive `long` that's why we used 0 for disabling,  Now that we are utilizing existing [BatchingSettings](https://github.com/googleapis/gax-java/blob/master/gax/src/main/java/com/google/api/gax/batching/BatchingSettings.java#L167-L177) which allows null or greater than 0 values only.

If we don't call any setter than the default values provided in the BatchingSettings will be used which is 1 element, 1 byte or 1 millisecond. Although each google-cloud-client will have their own default values.